pkg/controllers/cloud/networkpolicy.go

```diff
@@ -280,6 +280,14 @@ func vpcsFromGroupMembers(members []antreanetworking.GroupMember, r *NetworkPoli
 				continue
 			}
 			cloudRsc.Name.Name = cloudAssignedID
+
+			cloudAccountID, ok := ownerAnnotations[cloudcommon.AnnotationCloudAccountIDKey]
```

Comment:
nit: remove new line and use hash lookup convention. have ok in the same line

Reply:
fixed, can't have ok in the same line as cloudAccountID is used later in the code.